core/src/scene/drawRule.h

```diff
@@ -37,11 +38,10 @@ class StyleContext;
 struct DrawRuleData {
 
     std::vector<StyleParam> parameters;
-    std::string name;
-    int id;
+    std::string styleName;
+    int styleId;
```

Comment:
In general, the name of a draw rule is **not** the same as the name of the style used by the rule. If a style is not explicitly specified then it will be inferred from the name of the rule, but these things are _necessarily distinct_ and the names of variables should clearly reflect that.